custom/languages/Russian/time.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+/*
+ *	Made by Samerton
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0-dev
+ *
+ *  License: MIT
+ *
+ *  Russian Language - Time
+ */
+
+$language = array(
```

Comment:
Were you able to handle plurals correctly in translations such as this? There was a feature added over a year ago that should help if not: https://github.com/NamelessMC/Nameless/issues/1419#issuecomment-475904785

Reply:
Yes, it works great, I added it to my translation